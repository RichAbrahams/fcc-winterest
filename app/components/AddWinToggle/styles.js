import styled from 'styled-components';

export const Wrapper = styled.div `
  display: flex;
  align-items: center;
  justify-content: flex-end;
  width: 100%;
  padding: 0em 5em 0em 0em;
`;

export const Button = styled.button `
  padding: 0.5em  1em 0.5em  1em;
  background: white;
  border: 2px solid #444;
  margin: 0em 0em 0em 2em;
`;

