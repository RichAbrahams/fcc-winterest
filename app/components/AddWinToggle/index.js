/**
*
* AddWin
*
*/

import React from 'react';
import { Wrapper, Button } from './styles';


function AddWinToggle({ toggleAddWin }) {
  return (
    <Wrapper>
      <Button onClick={() => toggleAddWin()}>
        Add Win
      </Button>
    </Wrapper>
  );
}

AddWinToggle.propTypes = {

};

export default AddWinToggle;
