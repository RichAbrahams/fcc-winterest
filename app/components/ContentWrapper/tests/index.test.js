import React from 'react';
import renderer from 'react-test-renderer';
import AddWinToggle from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <AddWinToggle />).toJSON();
  expect(tree).toMatchSnapshot();
});
