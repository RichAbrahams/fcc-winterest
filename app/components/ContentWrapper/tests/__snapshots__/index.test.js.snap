exports[`test renders correctly 1`] = `
<div
  className="content-wrapper kzeqBg" />
`;
